### PR TITLE
DOC/examples: Rename folder "get_started" to "intro"

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR suggests to rename the folder "get_started" to "intro" to be more descriptive and match the title "Intro to PyGMT" of the examples in the documentation.

**Preview**: https://pygmt-dev--3571.org.readthedocs.build/en/3571/intro/index.html

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Comment by seisman** [comment]
I'm OK with the renaming. @GenericMappingTools/pygmt-maintainers What do you think?

File: doc/conf.py [hunk 1]
```diff
@@ -163,11 +163,11 @@
...
     "gallery/**.ipynb",
     "tutorials/**.ipynb",
-    "get_started/**.ipynb",
+    "intro/**.ipynb",
     "projections/**.ipynb",
     "gallery/**.py",
     "tutorials/**.py",
-    "get_started/**.py",
+    "intro/**.py",
     "projections/**.py",
```
Comment (seisman) [inline doc/conf.py:171]: Maybe sort them alphabetically or sort based on the order they appear in the doc?

File: doc/Makefile [hunk 2]
```diff
@@ -57,5 +57,5 @@ clean:
 	rm -rf api/generated
 	rm -rf gallery
 	rm -rf tutorials
-	rm -rf get_started
+	rm -rf intro
```
Comment (seisman) [inline doc/Makefile:60]: Also need to update `.gitignore`.

Comment (yvonnefroehlich) [reply doc/Makefile:60]: Updated in commit https://github.com/GenericMappingTools/pygmt/pull/3571/commits/d859020df5ddfa16312973a5c1330d443f5cd13e.

File: doc/conf.py (hunk 1 above)
Comment (seisman) [reply doc/conf.py:171]: > sort based on the order they appear in the doc?

I think this is better. I.e., "intro", "tutorials", "gallery", "projections".

Comment (yvonnefroehlich) [reply doc/conf.py]: See comment https://github.com/GenericMappingTools/pygmt/pull/3571/commits/7ce7855a68735dfba10a8a21e40cf9359667a836.

File: doc/conf.py [hunk 3]
```diff
@@ -105,11 +105,11 @@
     "examples_dirs": [
         "../examples/gallery",
```
Comment (seisman) [inline doc/conf.py:106]: Please also sort lines 106-110, lines 115-132, and line 112.

File: doc/conf.py [hunk 4]
```diff
@@ -105,11 +105,11 @@
...
         "../examples/tutorials",
-        "../examples/get_started",
+        "../examples/intro",
         "../examples/projections",
     ],
     # path where to save gallery generated examples
-    "gallery_dirs": ["gallery", "tutorials", "get_started", "projections"],
+    "gallery_dirs": ["gallery", "tutorials", "intro", "projections"],
```
Comment (seisman) [inline doc/conf.py:112]:
```suggestion
    "gallery_dirs": ["intro", "tutorials", "gallery", "projections"],
```

File: doc/conf.py (hunk 3 above)
Comment (yvonnefroehlich) [reply doc/conf.py]: See commit https://github.com/GenericMappingTools/pygmt/commit/e2995083de6e8bdbd0f1b3056f52263547965f07.

File: doc/conf.py (hunk 4 above)
Comment (yvonnefroehlich) [reply doc/conf.py]: See commit https://github.com/GenericMappingTools/pygmt/commit/e2995083de6e8bdbd0f1b3056f52263547965f07.

**Review by seisman — APPROVED** [review]

**Pull request merged**